build_tools/benchmarks/collect_compilation_statistics.py

```diff
@@ -184,11 +184,14 @@ def main(args: argparse.Namespace):
             f"Module path isn't a module cmake target: {module_path}")
```

Comment:
[Here for threading]

Sorry I missed this earlier.

I'm a bit concerned that we're juggling thing around to adapt to the natural keys. I was hoping we could do away with those in the near future.

Reply:
I think one way is to also upload the artificial keys to the dashboard, so it will be easier when we migrate data from natural keys to artificial keys.